src/core/render/compiler.js

```diff
@@ -57,6 +57,10 @@ const compileMedia = {
   },
 };
 
+/**
+ * This class compiles the markdown of each page of a Docsify site into HTML
+ * (using some information from the provided router).
+ */
```

Comment:
Good idea to add inline documentation me thinks. It shows up in the intellisense of your editor when hovering on the function. Make things easier to learn.